README.md

```diff
@@ -67,57 +64,57 @@ It's based on Rails 4 and Ruby 2.1.5.
 
 ## Getting Started
 
-Clone application as new project with original repository named "rails-base"
-
-    git clone --depth 1 git://github.com/fs/rails-base.git --origin rails-base [MY-NEW-PROJECT]
-
-**Note: we use depth parameter here in order not to copy the history of changes in base project**
+### Prepare dependencies
 
-Create your new repo on GitHub and push master into it.
-Make sure master branch is tracking origin repo.
+Application based on Ruby 2.1.5 and PostgreSQL, some gems have native extensions.
+You should have GCC installed on you development machine.
 
-    git remote add origin git@github.com:[MY-GITHUB-ACCOUNT]/[MY-NEW-PROJECT].git
-    git push -u origin master
+* `brew install qt` - to run spec with [Capybara Webkit](https://github.com/thoughtbot/capybara-webkit)
+* `brew install phantomjs198` - to run Javascript unit tests
 
-Run setup script
+### Bootstrap application
 
-    bin/setup
+1. Clone application as new project with original repository named "rails-base". We use depth parameter here in order not to copy the history of changes in base project
 
-Install phantomjs 1.9.8 to run Javascript unit tests:
+   ```bash
+   git clone --depth 1 git://github.com/fs/rails-base.git --origin rails-base [MY-NEW-PROJECT]
+   ```
 
-    brew install phantomjs198
+2. Create your new repo on GitHub and push master into it. Make sure master branch is tracking origin repo.
 
-Make sure all test are green
+  ```bash
+  git remote add origin git@github.com:[MY-GITHUB-ACCOUNT]/[MY-NEW-PROJECT].git
+  git push -u origin master
+  ```
 
-    bin/ci
+3. Run setup script
 
-Run app
+  ```bash
+  bin/setup
+  ```
 
-    bin/server
+4. Make sure all test are green
```

Comment:
to make sure all deps are satisfied and applications works correctly before making changes.